packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -619,18 +619,20 @@ export function isTransferConfirmed(tx: MergedTransaction) {
   return dayjs().isAfter(getConfirmedDate(tx))
 }
 
-export function useRemainingTime(tx: MergedTransaction) {
-  const [remainingTime, setRemainingTime] = useState<string>('Calculating...')
+export function useRemainingTimeCctp(tx: MergedTransaction) {
+  const [estimatedMinutesLeft, setEstimatedMinutesLeft] = useState<
+    number | null
+  >(null)
   const [canBeClaimedDate, setCanBeClaimedDate] = useState<dayjs.Dayjs>()
   const [isConfirmed, setIsConfirmed] = useState(
     tx.status === 'Confirmed' || tx.status === 'Executed'
   )
 
   useEffect(() => {
     if (tx.status === 'Failure') {
-      setRemainingTime('')
+      setEstimatedMinutesLeft(null)
```

Comment:
we need a unit test for this case
i don't see it atm

Reply:
We can't have hardcoded unit test because cctp fetches remaining time dynamically

Reply:
To clarify this more, we currently mock up a transaction with a lot of empty values just to get hardcoded remaining time. With cctp we wouldn't be able to fetch this time because the tx doesn't exist